lls/main.c

```diff
@@ -64,9 +116,24 @@ process_pkts(__attribute__((unused)) struct lls_config *lls_conf,
 			goto free_buf;
 
 		ether_type = eth_hdr->ether_type;
+		/*
+		 * Calculate offset to skip over VLAN header, if present.
+		 * TODO If VLAN stripping is enabled, we can skip this.
```

Comment:
This TODO is even bigger, we need to add VLAN headers on the interfaces that require it. The GK block isn't doing anything about it.